src/weather_station/clima_sensor_us.rs

```diff
@@ -0,0 +1,184 @@
+extern crate libmodbus;
+use libmodbus::*;
+
+///Constants
+const DEVICE: &'static str = "/dev/ttyS5";
+const BAUDRATE: i32 = 9600;
+const PARITY: char = 'N';
+const DATA_BITS: i32 = 8;
+const STOP_BITS: i32 = 1;
+const SLAVE_ID: u8 = 1;
+
+const ERROR_CODE_S32: u32 = 0x7FFFFFFF;
+const ERROR_CODE_U32: u32 = 0xFFFFFFFF;
+
+//Special reg-addresses
```

Comment:
They are not special, are they?

```suggestion
// Addresses of registers.
```